docker/flink/docker-entrypoint.sh

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env bash
```

Comment:
```suggestion
#!/usr/bin/env bash
# shellcheck disable=all
```

Reply:
Why disable this?

Reply:
> Why disable this?

Sorry, this is my first time seeing that comment. I'm not sure how it got resolved; wasn't intentional in any case. :sweat_smile:

I just disabled it for testing purposes before a flight, because I wasn't sure the solution would work and just wanted CI to run.

> I think it'll be fine to leave `shellcheck` disabled on the added file, I'll just add a comment as to where it's copied from

It's simply copied from https://github.com/apache/flink-docker/blob/206aa67/1.19/scala_2.12-java11-ubuntu/Dockerfile, which is why I think it may make sense to just note that, instead of adding `shellcheck` (so that, if anybody were to diff it against the original file, in case something changes, it would be more obvious). Otherwise, I can update it to pass `shellcheck`; I wanted to check the approach before wasting time on that.